### PR TITLE
[PATCH v6] Speed up make check

### PR DESCRIPTION
**Pull request by JannePeltonen**
Make time validation test and timer_accuracy example complete faster to speed up make check

### CODE REVIEW

File: test/validation/api/time/time.c [hunk 1]
```diff
@@ -405,20 +407,20 @@ static void time_test_wait_until(time_cb time_cur, time_from_ns_cb time_from_ns)
 	int i;
 	odp_time_t lower_limit, upper_limit;
 	odp_time_t start_time, end_time, wait;
-	odp_time_t second = time_from_ns(ODP_TIME_SEC_IN_NS);
+	odp_time_t second = time_from_ns(WAIT_TIME);
```
[the remark this answers is not in the document]
Comment (JannePeltonen) [reply test/validation/api/time/time.c]: Renamed in v2, also in an another function.

**Review by TuomasTaipale — APPROVED** [review]

**Comment by JannePeltonen** [comment]
v4: Added missing EXTRA_DIST for the added shell script.

**Review by TuomasTaipale — APPROVED** [review]

File: test/validation/api/time/time.c [hunk 2]
```diff
@@ -543,54 +564,16 @@ static void time_test_accuracy_nsec(time_nsec_cb time_nsec)
...
-	time_test_accuracy_nsec(odp_time_local_strict_ns);
-}
-
-static void time_test_global_strict_accuracy_nsec(void)
-{
-	time_test_accuracy_nsec(odp_time_global_strict_ns);
+		/* Check that ODP time is within +-5% of system time */
+		CU_ASSERT(sec_t < sec_c * 1.05);
+		CU_ASSERT(sec_t > sec_c * 0.95);
+	}
```
Comment (psavol) [inline test/validation/api/time/time.c:576]: When one test fails with a loop, it's hard to tell from a log which one failed. Maybe unroll the loop, or print out something on a failure. Also the loop on line 520 could be changed the same way.

**Comment by JannePeltonen** [comment]
v5:
- added the requested printouts to identify failed tests better
- added a new patch to remove a redundant part of time monotony test to make it run faster

**Review by TuomasTaipale — APPROVED** [review]
One commit-message related comment, otherwise looks OK to me.

**Comment by JannePeltonen** [comment]
v6: Fixed the typo and added reviewed-by tags.

**Pull request merged**